src/app/login/login.component.css

```diff
@@ -0,0 +1,200 @@
+* {

```

Comment:
I have a feeling that we have too many styles in this file and we don't use all of them. At least I don't see any inputs with type number on our page.

Can you please remove the extra code?